matterway/verdaccio-auth0-ui
# Support verdaccio v5

How to test it:
1. checkout this branch and build `npm run build`
1. Install verdaccio globally `npm install --location=global verdaccio`
1. Create new project (simply make new directory and open it)
1. Install dependencies, run `npm install verdaccio@5 verdaccio-auth0-ui verdaccio-aws-s3-storage verdaccio-github-oauth --force`
1. create config.yaml (take it from registry server or contact @aju)
1. replace `dist` from `node_modules/verdaccio-auth0-ui` with `dist` folder from this project.
1. run registry server `verdaccio -c config.yaml`
1. set your registry to use local server, go to http://localhost:4873/, login and next open configuration <img width="1269" alt="Screenshot 2022-10-07 at 12 58 20" src="https://user-images.githubusercontent.com/316313/194538763-1dea711a-85ea-4670-901b-0b0b0ae0dcb3.png">
1. run commands to setup registry 
<img width="643" alt="Screenshot 2022-10-07 at 12 58 27" src="https://user-images.githubusercontent.com/316313/194538913-b8542ba7-6200-43ef-bce8-ca58d4430eba.png">

Now you can use this local registry to install @matterway packages, for example creating new skill from template:
```
yarn create @matterway/assistant-skill --template @matterway/assistant-skill-template-v10 skill-name
```
when installing you should see logs in verdaccio server
